template.go

```diff
@@ -18,8 +18,14 @@ import (
 var (
 	//go:embed template
 	templateDir embed.FS
+)
+
+func genTemplates(cfg *Config) *gen.Template {
 	// funcMap contains extra template functions used by ogent.
-	funcMap = template.FuncMap{
+	funcMap := template.FuncMap{
+		"allowClientUUIDs": func() bool {
+			return cfg.AllowClientUUIDs
+		},
```

Comment:
Instead of this I prefer to use `extend` and add the flag there. Have a look at how it is done inside Ent: https://github.com/ent/ent/blob/3f1063c77ec98d1b1d5d1cdc39659cd13931ab8f/entc/gen/func.go#L265